ui/docs/reference/workers/generic-worker/installing.mdx

```diff
@@ -156,43 +160,88 @@ __These instructions require macOS Mojave version 10.14.x__
       CheckForNetwork
     done
 
+    cd ~
+
     /usr/local/bin/generic-worker run --config /etc/generic-worker/config
     ```
 
- 9. Run the following to make the `run-generic-worker.sh` script executable:
+ 10. Run the following to make the `run-generic-worker.sh` script executable:
 
     * `chmod a+x /usr/local/bin/run-generic-worker.sh`
 
-10. Create `/etc/generic-worker/config` with appropriate configuration settings
+11. Create `/etc/generic-worker/config` with appropriate configuration settings
     (see `generic-worker --help` for details). Something like this:
 
     ```
     {
-      "accessToken": "********************************************",
-      "cachesDir": "/Library/Caches/generic-worker/caches",
-      "cleanUpTaskDirs": true,
-      "clientId": "project/foo/bar/my/client",
-      "disableReboots": false,
-      "downloadsDir": "/Library/Caches/generic-worker/downloads",
-      "ed25519SigningKeyLocation": "/etc/generic-worker/ed25519_key",
-      "workerPoolId": "my/worker-pool",
-      "publicIP": "1.2.3.4",
-      "requiredDiskSpaceMegabytes": 2048,
-      "rootURL": "https://my-root-url.com",
-      "sentryProject": "generic-worker",
-      "shutdownMachineOnIdle": false,
-      "shutdownMachineOnInternalError": false,
-      "subdomain": "taskcluster-worker.net",
-      "taskclusterProxyPort": 8080,
-      "tasksDir": "/Users",
-      "workerGroup": "my-worker-group",
-      "workerId": "my-lovely-computer",
-      "wstAudience": "taskcluster-net",
-      "wstServerURL": "https://websocktunnel.tasks.build"
-    }
-    ```
-
-11. Create launch daemon:
+     "accessToken": "********************************************",
+     "cachesDir": "/Library/Caches/generic-worker/caches",
+     "cleanUpTaskDirs": true,
+     "clientId": "project/releng/generic-worker/datacenter-gecko-t-osx-1400-m2-staging",
+     "disableReboots": false,
+     "downloadsDir": "/Library/Caches/generic-worker/downloads",
+     "ed25519SigningKeyLocation": "/etc/generic-worker/ed25519_key",
+     "livelogExecutable": "/usr/local/bin/livelog",
+     "provisionerId": "releng-hardware",
+     "publicIP": "192.168.5.174",
+     "requiredDiskSpaceMegabytes": 20480,
+     "rootURL": "https://firefox-ci-tc.services.mozilla.com",
+     "runTasksAsCurrentUser": true,
+     "sentryProject": "generic-worker",
+     "shutdownMachineOnIdle": false,
+     "shutdownMachineOnInternalError": false,
+     "taskclusterProxyExecutable": "/usr/local/bin/taskcluster-proxy",
+     "taskclusterProxyPort": 8080,
+     "tasksDir": "/Users",
+     "workerGroup": "macstadium-vegas",
+     "workerId": "moz-runner-multi-1",
+     "workerLocation": "{\"cloud\":\"standalone\",\"host\":\"macstadium\"}",
+     "workerType": "gecko-t-osx-1400-m2-staging",
+     "workerTypeMetadata": {},
+     "wstAudience": "firefoxcitc",
+     "wstServerURL": "https://firefoxci-websocktunnel.services.mozilla.com/"
+  }
+    ```
```

Comment:
This worker config can all be written in the worker runner (`/etc/generic-worker/runner.yml`) config file under `workerConfig` and this file will be written out by `start-worker`. I think it'd be safe to remove this point.